components/esp_websocket_client/esp_websocket_client.c

```diff
@@ -1174,7 +1174,6 @@ static void esp_websocket_client_task(void *pv)
                 ESP_LOGV(TAG, "Read poll timeout: skipping esp_transport_read()...");
                 break;
             }
-            client->ping_tick_ms = _tick_get_ms();
```

Comment:
Question: Should we make it configurable? I mean it makes sense for some usecases -- saves unnecessary PING traffic when sock is readable?

Reply:
However, if you think the "skip PING during traffic" behavior has value, I can add explicit `skip_ping_on_traffic` configuration option

Reply:
@david-cermak thank you for the review! I will push it as it is now. If users report issues with PING traffic during active connections, then will add it in a seperate PR